README.md

```diff
@@ -3,12 +3,109 @@
 ## Install and run locally
 
 You can install the package with `pip install .` from the repository's root.
-Alternatively you can use [Poetry](https://python-poetry.org/docs/basic-usage/#installing-dependencies) to install and manage a virtual environment.
+Alternatively you can use [Poetry](https://python-poetry.org/docs/basic-usage/#installing-dependencies)
+to install and manage a virtual environment.
 
 Then set two environment variables:
 
- - `DB_URL`: currently refers to the mlwh's url, in the format: `mysql+pymysql://user:pass@host:port/dbname`
- - `CORS_ORIGINS`: a comma-separated list of origins to allow CORS, e.g: `http://localhost:3000,https://example.com:443`
+- `DB_URL`: currently refers to the mlwh's url, in the format: `mysql+pymysql://user:pass@host:port/dbname`
+- `CORS_ORIGINS`: a comma-separated list of origins to allow CORS, e.g: `http://localhost:3000,https://example.com:443`
 
-Finally, run the server: `uvicorn lang_qc.main:app`. 
+Finally, run the server: `uvicorn lang_qc.main:app`.
 Or `uvicorn lang_qc.main:app --reload` to reload the server on code changes.
+
+## Using Docker
+
+This repository provides a [Docker image](https://github.com/wtsi-npg/npg_langqc/pkgs/container/npg_langqc).
+The image is configured to use SSL for the server, running uvicorn.
+
+The following instructions assume you have a recent Ubuntu system with Docker installed,
+and your desired ports open.
+
+### Step 1
+
+Copy over SSL certificates, put them in a folder and name or symlink them to `ca.pem`,
+`key.pem` and `cert.pem`.
```

Comment:
name or symlink them to `ca.pem`, `key.pem` and `cert.pem` ->

name them `ca.pem`, `key.pem` and `cert.pem` or create symlinks to them using these names.